src/markers/utils.js

```diff
@@ -53,3 +53,11 @@ export const concatenateUint8Arrays = (arrays) => {
 
   return concatenatedArray;
 };
+
+export const readInt32BE = (array, offset) => {
+  return readUInt32BE(array, offset) | 0;
+};
+
+export const readInt32LE = (array, offset) => {
+  return readUInt32LE(array, offset) | 0;
+};
```

Comment:
Why are these needed?

Also please add a new line at the end of the file :)

Reply:
These are for handling Case 9 & 10. Case 9 & 10 involve SLONG. EXIF standard describes SLONG as

"A 32-bit (4-byte) signed integer (2's complement notation)."

Hence I added these methods to read signed ints, as previous methods only read unsigned numbers.